lib/scripts/get_tags.py

```diff
@@ -37,13 +37,8 @@
 if user_tags:
     try:
         user_tags = user_tags.split(',')
-        for tag in user_tags:
-            tags.append(tag)
+        tags.extend(user_tags)
     except Exception as e:
         print("there was an issue parsing the tags in DD_TAGS: {}".format(e))
 
-legacy_tags = os.environ.get('LEGACY_TAGS_FORMAT', False)
-if legacy_tags:
-    print(','.join(tags))
-else:
-    print(json.dumps(tags))
+print(','.join(tags))
```

Comment:
What was that and why don't we need it anymore?

Reply:
We used to write to the datadog.yaml, with `sed`, and this needed to be an array.
I feel like this is not the proper way to do it, as it is very sensitive to datadog.yaml changes (e.g. refactor to follow parameter description convention).
Everything is configured through environment variables now, and the env var for tags takes a comma separated list of tags.